modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/services/rest/TestrayTask.ts

```diff
@@ -64,6 +64,44 @@ class TestrayTaskImpl extends Rest<TaskForm, TestrayTask, NestedObjectOptions> {
 		});
 	}
 
+	private async getTaskUsers(taskId: number) {
+		const taskUsersResponse = await testrayTaskUsersImpl.getAll(
+			searchUtil.eq('taskId', taskId)
+		);
+
+		if (!taskUsersResponse) {
+			return [];
+		}
+
+		const taskUsers =
+			testrayTaskUsersImpl.transformDataFromList(taskUsersResponse)
+				?.items || [];
+
+		return taskUsers.map(({id}) => id);
+	}
```

Comment:
This entire function returns an array of number, we could 

rename the function to `getTaskUserIds`

Since you are returning a number[]

There's no need to transform the data, since you don't need anything, just the id.

```suggestion
	private async getTaskUserIds(taskId: number) {
		const response = await testrayTaskUsersImpl.getAll(
			searchUtil.eq('taskId', taskId)
		);

		return (response?.items ?? [])?.map(({id}) => id);
	}
```